### ISSUE TITLE
gzip: ...: Too many levels of symbolic links

### ISSUE THREAD

**Issue by ChristophKnapp** [nf-core/mag#327]
### Description of the bug

Hello,
when I try to run nf-core I run into the following problem. Its trying to unzip the SPAdes-NG-30689_QN1_4_3_lib613328_10075_2-depth.txt.gz archive. and stops with gzip: ...: Too many levels of symbolic links. 

I tried to add the -f flag in the .command.sh scripts and that helped to get the analysis about halfway through. Now I'm at a point where it is happening every time with SPADES and MEGAHIT.

It seems every time I resume the analysis it creates a new workdirectory where it runs into the same problem.  

I run nf-core with the following command.

nextflow run nf-core/mag -profile conda --input '../data/*_R{1,2}.fastq.gz' --outdir results -resume

I had trouble setting up docker, singularity and the other environments. Conda was the only one I could get to work somehow. So this might be a Problem with my setup. 

Thanks for your help

Christoph

### Command used and terminal output

```console
nextflow run nf-core/mag -profile conda --input '../data/*_R{1,2}.fastq.gz' --outdir results -resume
N E X T F L O W  ~  version 22.04.5
Launching `https://github.com/nf-core/mag` [zen_mirzakhani] DSL2 - revision: 3b4dd34697 [master]


------------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/mag v2.2.0
------------------------------------------------------
Core Nextflow options
  revision        : master
  runName         : zen_mirzakhani
  launchDir       : /home/hummelchen/bioinformatik/NGS/nf-core-workflow
  workDir         : /home/hummelchen/bioinformatik/NGS/nf-core-workflow/work
  projectDir      : /home/hummelchen/.nextflow/assets/nf-core/mag
  userName        : hummelchen
  profile         : conda
  configFiles     : /home/hummelchen/.nextflow/assets/nf-core/mag/nextflow.config

Input/output options
  input           : ../data/*_R{1,2}.fastq.gz
  outdir          : results

Generic options
  enable_conda    : true

Quality control for short reads options
  phix_reference  : /home/hummelchen/.nextflow/assets/nf-core/mag/assets/data/GCA_002596845.1_ASM259684v1_genomic.fna.gz

Quality control for long reads options
  lambda_reference: /home/hummelchen/.nextflow/assets/nf-core/mag/assets/data/GCA_000840245.1_ViralProj14204_genomic.fna.gz

Taxonomic profiling options
  gtdb            : https://data.ace.uq.edu.au/public/gtdb/data/releases/release202/202.0/auxillary_files/gtdbtk_r202_data.tar.gz

!! Only displaying parameters that differ from the pipeline defaults !!
------------------------------------------------------
If you use nf-core/mag for your analysis please cite:

* The pipeline publication
  https://doi.org/10.1093/nargab/lqac007

* The pipeline
  https://doi.org/10.5281/zenodo.3589527

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x

* Software dependencies
  https://github.com/nf-core/mag/blob/master/CITATIONS.md
------------------------------------------------------
executor >  local (11)
[43/6c36ea] process > NFCORE_MAG:MAG:FASTQC_RAW (NG-30689_QN1_4_3_lib613328_10075_2)                                                                            [100%] 1 of 1, cached: 1 ✔
[ac/b3eb56] process > NFCORE_MAG:MAG:FASTP (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                 [100%] 1 of 1, cached: 1 ✔
[fc/8581e8] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_BUILD (GCA_002596845.1_ASM259684v1_genomic.fna.gz)                                                    [100%] 1 of 1, cached: 1 ✔
[af/ae4ac2] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_ALIGN (NG-30689_QN1_4_3_lib613328_10075_2)                                                            [100%] 1 of 1, cached: 1 ✔
[c0/e3e946] process > NFCORE_MAG:MAG:FASTQC_TRIMMED (NG-30689_QN1_4_3_lib613328_10075_2)                                                                        [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_RAW                                                                                                               -
[-        ] process > NFCORE_MAG:MAG:PORECHOP                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOLYSE                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:FILTLONG                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_FILTERED                                                                                                          -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE_DB_PREPARATION                                                                                                  -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2_DB_PREPARATION                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2                                                                                                                    -
[73/0757dd] process > NFCORE_MAG:MAG:MEGAHIT (NG-30689_QN1_4_3_lib613328_10075_2)                                                                               [100%] 1 of 1, cached: 1 ✔
[f5/c2b808] process > NFCORE_MAG:MAG:SPADES (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:SPADESHYBRID                                                                                                               -
[e5/23bfe7] process > NFCORE_MAG:MAG:QUAST (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                                                          [100%] 2 of 2, cached: 2 ✔
[1c/647ef7] process > NFCORE_MAG:MAG:PRODIGAL (NG-30689_QN1_4_3_lib613328_10075_2)                                                                              [100%] 2 of 2, cached: 2 ✔
[76/706846] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_BUILD (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                     [100%] 2 of 2, cached: 2 ✔
[34/af10cf] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_ALIGN (MEGAHIT-NG-30689_QN1_4_3_lib613328_10075_2-NG-30689_QN1_4_3_lib613328_10075_2) [100%] 2 of 2, cached: 2 ✔
[8d/3bfe97] process > NFCORE_MAG:MAG:BINNING:METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                          [100%] 2 of 2, cached: 2 ✔
[5e/10afc4] process > NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                                                [  0%] 0 of 2
[9a/2903bc] process > NFCORE_MAG:MAG:BINNING:METABAT2_METABAT2 (NG-30689_QN1_4_3_lib613328_10075_2)                                                             [100%] 2 of 2, cached: 2 ✔
executor >  local (11)
[43/6c36ea] process > NFCORE_MAG:MAG:FASTQC_RAW (NG-30689_QN1_4_3_lib613328_10075_2)                                                                            [100%] 1 of 1, cached: 1 ✔
[ac/b3eb56] process > NFCORE_MAG:MAG:FASTP (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                 [100%] 1 of 1, cached: 1 ✔
[fc/8581e8] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_BUILD (GCA_002596845.1_ASM259684v1_genomic.fna.gz)                                                    [100%] 1 of 1, cached: 1 ✔
[af/ae4ac2] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_ALIGN (NG-30689_QN1_4_3_lib613328_10075_2)                                                            [100%] 1 of 1, cached: 1 ✔
[c0/e3e946] process > NFCORE_MAG:MAG:FASTQC_TRIMMED (NG-30689_QN1_4_3_lib613328_10075_2)                                                                        [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_RAW                                                                                                               -
[-        ] process > NFCORE_MAG:MAG:PORECHOP                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOLYSE                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:FILTLONG                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_FILTERED                                                                                                          -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE_DB_PREPARATION                                                                                                  -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2_DB_PREPARATION                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2                                                                                                                    -
[73/0757dd] process > NFCORE_MAG:MAG:MEGAHIT (NG-30689_QN1_4_3_lib613328_10075_2)                                                                               [100%] 1 of 1, cached: 1 ✔
[f5/c2b808] process > NFCORE_MAG:MAG:SPADES (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:SPADESHYBRID                                                                                                               -
[e5/23bfe7] process > NFCORE_MAG:MAG:QUAST (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                                                          [100%] 2 of 2, cached: 2 ✔
[1c/647ef7] process > NFCORE_MAG:MAG:PRODIGAL (NG-30689_QN1_4_3_lib613328_10075_2)                                                                              [100%] 2 of 2, cached: 2 ✔
[76/706846] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_BUILD (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                     [100%] 2 of 2, cached: 2 ✔
[34/af10cf] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_ALIGN (MEGAHIT-NG-30689_QN1_4_3_lib613328_10075_2-NG-30689_QN1_4_3_lib613328_10075_2) [100%] 2 of 2, cached: 2 ✔
[8d/3bfe97] process > NFCORE_MAG:MAG:BINNING:METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                          [100%] 2 of 2, cached: 2 ✔
[12/d40e4b] process > NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                                                [100%] 2 of 2, failed: 2 ✘
[9a/2903bc] process > NFCORE_MAG:MAG:BINNING:METABAT2_METABAT2 (NG-30689_QN1_4_3_lib613328_10075_2)                                                             [100%] 2 of 2, cached: 2 ✔
[-        ] process > NFCORE_MAG:MAG:BINNING:MAXBIN2                                                                                                            -
executor >  local (11)
[43/6c36ea] process > NFCORE_MAG:MAG:FASTQC_RAW (NG-30689_QN1_4_3_lib613328_10075_2)                                                                            [100%] 1 of 1, cached: 1 ✔
[ac/b3eb56] process > NFCORE_MAG:MAG:FASTP (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                 [100%] 1 of 1, cached: 1 ✔
[fc/8581e8] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_BUILD (GCA_002596845.1_ASM259684v1_genomic.fna.gz)                                                    [100%] 1 of 1, cached: 1 ✔
[af/ae4ac2] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_ALIGN (NG-30689_QN1_4_3_lib613328_10075_2)                                                            [100%] 1 of 1, cached: 1 ✔
[c0/e3e946] process > NFCORE_MAG:MAG:FASTQC_TRIMMED (NG-30689_QN1_4_3_lib613328_10075_2)                                                                        [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_RAW                                                                                                               -
[-        ] process > NFCORE_MAG:MAG:PORECHOP                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOLYSE                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:FILTLONG                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_FILTERED                                                                                                          -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE_DB_PREPARATION                                                                                                  -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2_DB_PREPARATION                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2                                                                                                                    -
[73/0757dd] process > NFCORE_MAG:MAG:MEGAHIT (NG-30689_QN1_4_3_lib613328_10075_2)                                                                               [100%] 1 of 1, cached: 1 ✔
[f5/c2b808] process > NFCORE_MAG:MAG:SPADES (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:SPADESHYBRID                                                                                                               -
[e5/23bfe7] process > NFCORE_MAG:MAG:QUAST (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                                                          [100%] 2 of 2, cached: 2 ✔
[1c/647ef7] process > NFCORE_MAG:MAG:PRODIGAL (NG-30689_QN1_4_3_lib613328_10075_2)                                                                              [100%] 2 of 2, cached: 2 ✔
[76/706846] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_BUILD (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                     [100%] 2 of 2, cached: 2 ✔
[34/af10cf] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_ALIGN (MEGAHIT-NG-30689_QN1_4_3_lib613328_10075_2-NG-30689_QN1_4_3_lib613328_10075_2) [100%] 2 of 2, cached: 2 ✔
[8d/3bfe97] process > NFCORE_MAG:MAG:BINNING:METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                          [100%] 2 of 2, cached: 2 ✔
[12/d40e4b] process > NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                                                [100%] 2 of 2, failed: 2 ✘
[9a/2903bc] process > NFCORE_MAG:MAG:BINNING:METABAT2_METABAT2 (NG-30689_QN1_4_3_lib613328_10075_2)                                                             [100%] 2 of 2, cached: 2 ✔
[-        ] process > NFCORE_MAG:MAG:BINNING:MAXBIN2                                                                                                            -
[-        ] process > NFCORE_MAG:MAG:BINNING:ADJUST_MAXBIN2_EXT                                                                                                 -
executor >  local (11)
[43/6c36ea] process > NFCORE_MAG:MAG:FASTQC_RAW (NG-30689_QN1_4_3_lib613328_10075_2)                                                                            [100%] 1 of 1, cached: 1 ✔
[ac/b3eb56] process > NFCORE_MAG:MAG:FASTP (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                 [100%] 1 of 1, cached: 1 ✔
[fc/8581e8] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_BUILD (GCA_002596845.1_ASM259684v1_genomic.fna.gz)                                                    [100%] 1 of 1, cached: 1 ✔
[af/ae4ac2] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_ALIGN (NG-30689_QN1_4_3_lib613328_10075_2)                                                            [100%] 1 of 1, cached: 1 ✔
[c0/e3e946] process > NFCORE_MAG:MAG:FASTQC_TRIMMED (NG-30689_QN1_4_3_lib613328_10075_2)                                                                        [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_RAW                                                                                                               -
[-        ] process > NFCORE_MAG:MAG:PORECHOP                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOLYSE                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:FILTLONG                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_FILTERED                                                                                                          -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE_DB_PREPARATION                                                                                                  -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2_DB_PREPARATION                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2                                                                                                                    -
[73/0757dd] process > NFCORE_MAG:MAG:MEGAHIT (NG-30689_QN1_4_3_lib613328_10075_2)                                                                               [100%] 1 of 1, cached: 1 ✔
[f5/c2b808] process > NFCORE_MAG:MAG:SPADES (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:SPADESHYBRID                                                                                                               -
[e5/23bfe7] process > NFCORE_MAG:MAG:QUAST (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                                                          [100%] 2 of 2, cached: 2 ✔
[1c/647ef7] process > NFCORE_MAG:MAG:PRODIGAL (NG-30689_QN1_4_3_lib613328_10075_2)                                                                              [100%] 2 of 2, cached: 2 ✔
[76/706846] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_BUILD (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                     [100%] 2 of 2, cached: 2 ✔
[34/af10cf] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_ALIGN (MEGAHIT-NG-30689_QN1_4_3_lib613328_10075_2-NG-30689_QN1_4_3_lib613328_10075_2) [100%] 2 of 2, cached: 2 ✔
[8d/3bfe97] process > NFCORE_MAG:MAG:BINNING:METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                          [100%] 2 of 2, cached: 2 ✔
[12/d40e4b] process > NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                                                [100%] 2 of 2, failed: 2 ✘
[9a/2903bc] process > NFCORE_MAG:MAG:BINNING:METABAT2_METABAT2 (NG-30689_QN1_4_3_lib613328_10075_2)                                                             [100%] 2 of 2, cached: 2 ✔
[-        ] process > NFCORE_MAG:MAG:BINNING:MAXBIN2                                                                                                            -
[-        ] process > NFCORE_MAG:MAG:BINNING:ADJUST_MAXBIN2_EXT                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:BINNING:SPLIT_FASTA                                                                                                        [  0%] 0 of 2
executor >  local (11)
[43/6c36ea] process > NFCORE_MAG:MAG:FASTQC_RAW (NG-30689_QN1_4_3_lib613328_10075_2)                                                                            [100%] 1 of 1, cached: 1 ✔
[ac/b3eb56] process > NFCORE_MAG:MAG:FASTP (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                 [100%] 1 of 1, cached: 1 ✔
[fc/8581e8] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_BUILD (GCA_002596845.1_ASM259684v1_genomic.fna.gz)                                                    [100%] 1 of 1, cached: 1 ✔
[af/ae4ac2] process > NFCORE_MAG:MAG:BOWTIE2_PHIX_REMOVAL_ALIGN (NG-30689_QN1_4_3_lib613328_10075_2)                                                            [100%] 1 of 1, cached: 1 ✔
[c0/e3e946] process > NFCORE_MAG:MAG:FASTQC_TRIMMED (NG-30689_QN1_4_3_lib613328_10075_2)                                                                        [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_RAW                                                                                                               -
[-        ] process > NFCORE_MAG:MAG:PORECHOP                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOLYSE                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:FILTLONG                                                                                                                   -
[-        ] process > NFCORE_MAG:MAG:NANOPLOT_FILTERED                                                                                                          -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE_DB_PREPARATION                                                                                                  -
[-        ] process > NFCORE_MAG:MAG:CENTRIFUGE                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2_DB_PREPARATION                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:KRAKEN2                                                                                                                    -
[73/0757dd] process > NFCORE_MAG:MAG:MEGAHIT (NG-30689_QN1_4_3_lib613328_10075_2)                                                                               [100%] 1 of 1, cached: 1 ✔
[f5/c2b808] process > NFCORE_MAG:MAG:SPADES (NG-30689_QN1_4_3_lib613328_10075_2)                                                                                [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:SPADESHYBRID                                                                                                               -
[e5/23bfe7] process > NFCORE_MAG:MAG:QUAST (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                                                          [100%] 2 of 2, cached: 2 ✔
[1c/647ef7] process > NFCORE_MAG:MAG:PRODIGAL (NG-30689_QN1_4_3_lib613328_10075_2)                                                                              [100%] 2 of 2, cached: 2 ✔
[76/706846] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_BUILD (SPAdes-NG-30689_QN1_4_3_lib613328_10075_2)                                     [100%] 2 of 2, cached: 2 ✔
[34/af10cf] process > NFCORE_MAG:MAG:BINNING_PREPARATION:BOWTIE2_ASSEMBLY_ALIGN (MEGAHIT-NG-30689_QN1_4_3_lib613328_10075_2-NG-30689_QN1_4_3_lib613328_10075_2) [100%] 2 of 2, cached: 2 ✔
[8d/3bfe97] process > NFCORE_MAG:MAG:BINNING:METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                          [100%] 2 of 2, cached: 2 ✔
[12/d40e4b] process > NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)                                                                [100%] 2 of 2, failed: 2 ✘
[9a/2903bc] process > NFCORE_MAG:MAG:BINNING:METABAT2_METABAT2 (NG-30689_QN1_4_3_lib613328_10075_2)                                                             [100%] 2 of 2, cached: 2 ✔
[-        ] process > NFCORE_MAG:MAG:BINNING:MAXBIN2                                                                                                            -
[-        ] process > NFCORE_MAG:MAG:BINNING:ADJUST_MAXBIN2_EXT                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:BINNING:SPLIT_FASTA                                                                                                        [  0%] 0 of 2
[85/dedae7] process > NFCORE_MAG:MAG:BINNING:GUNZIP_BINS (SPAdes-MetaBAT2-NG-30689_QN1_4_3_lib613328_10075_2.20.fa.gz)                                          [ 79%] 46 of 58, cached: 37
[-        ] process > NFCORE_MAG:MAG:BINNING:GUNZIP_UNBINS                                                                                                      -
[-        ] process > NFCORE_MAG:MAG:BINNING:MAG_DEPTHS                                                                                                         -
[-        ] process > NFCORE_MAG:MAG:BINNING:MAG_DEPTHS_PLOT                                                                                                    -
[-        ] process > NFCORE_MAG:MAG:BINNING:MAG_DEPTHS_SUMMARY                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:BUSCO_QC:BUSCO                                                                                                             -
[-        ] process > NFCORE_MAG:MAG:BUSCO_QC:BUSCO_PLOT                                                                                                        -
[-        ] process > NFCORE_MAG:MAG:BUSCO_QC:BUSCO_SUMMARY                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:QUAST_BINS                                                                                                                 -
[-        ] process > NFCORE_MAG:MAG:QUAST_BINS_SUMMARY                                                                                                         -
[-        ] process > NFCORE_MAG:MAG:CAT                                                                                                                        -
[ec/0a99bc] process > NFCORE_MAG:MAG:GTDBTK:GTDBTK_DB_PREPARATION (gtdbtk_r202_data.tar.gz)                                                                     [100%] 1 of 1, cached: 1 ✔
[-        ] process > NFCORE_MAG:MAG:GTDBTK:GTDBTK_CLASSIFY                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:GTDBTK:GTDBTK_SUMMARY                                                                                                      -
[-        ] process > NFCORE_MAG:MAG:BIN_SUMMARY                                                                                                                -
[-        ] process > NFCORE_MAG:MAG:PROKKA                                                                                                                     -
[-        ] process > NFCORE_MAG:MAG:CUSTOM_DUMPSOFTWAREVERSIONS                                                                                                -
[-        ] process > NFCORE_MAG:MAG:MULTIQC                                                                                                                    -
Execution cancelled -- Finishing pending tasks before exit
-[nf-core/mag] Pipeline completed with errors-
WARN: Failed to render execution report -- see the log file for details
Error executing process > 'NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)'

Caused by:
  Process `NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS (NG-30689_QN1_4_3_lib613328_10075_2)` terminated with an error exit status (1)

Command executed:

  gunzip SPAdes-NG-30689_QN1_4_3_lib613328_10075_2-depth.txt.gz
  bioawk -t '{ { if (NR > 1) { { print $1, $3 } } } }' SPAdes-NG-30689_QN1_4_3_lib613328_10075_2-depth.txt > NG-30689_QN1_4_3_lib613328_10075_2_mb2_depth.txt
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_MAG:MAG:BINNING:CONVERT_DEPTHS":
      bioawk: $(bioawk --version | cut -f 3 -d ' ' )
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  gzip: SPAdes-NG-30689_QN1_4_3_lib613328_10075_2-depth.txt.gz: Too many levels of symbolic links

Work dir:
  /home/hummelchen/bioinformatik/NGS/nf-core-workflow/work/5e/10afc4d2fd48ad4b448fda9f96b88e

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`
```

### System information

N E X T F L O W  ~  version 22.04.5
nf-core/mag v2.2.0
Container engine: conda
OS: 
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster

Hardware: desktop with 128 Gb RAM and 32 cores

**Comment by apeltzer** [comment]
Adding the `-f` option is the way to solve this directly usually. Quite an umber of pipelines do that - if you have to modify the `.command.sh` yourself, this means it will not be automatically done on resuming a pipeline --> exactly the behaviour you see. 

Someone from team MAG should add that as a bugfix in a PR - that should make this permanently fixed. @skrakau maybe?

**Comment by jfy133** [comment]
@ChristophKnapp can you try `nextflow pull nf-core/mag`

and then run your run command but with `-r fix-convert-depths-gzip`? This follows the suggestion from @apeltzer above

**Comment by jfy133** [comment]
@ChristophKnapp due to another important bug fix, we are planning a new release tomorrow. If you could check today/early tomorrow if the linked branch works that would be great

**Comment by jfy133** [comment]
Ok going for a release, please reopen if the new release doesn't fix your issue

**Comment by ChristophKnapp** [comment]
I did as asked and that is the result.

$ nextflow pull nf-core/mag
Checking nf-core/mag ...
 Already-up-to-date - revision: 1b4456d542 [fix-convert-depths-gzip]

However I'm getting a different error now. I decided to start the analysis new to have a clean start.

I'm not sure this is fixed because of the other error. Because the other error does not seem related I will open a new issue.

**Comment by jfy133** [comment]
```
$ nextflow pull nf-core/mag
Checking nf-core/mag ...
Already-up-to-date - revision: https://github.com/nf-core/mag/commit/1b4456d542367cec093d62ff80d2ced8f65f25fe [fix-convert-depths-gzip]
```

This is just pulling, you would also need to run the pipeline.

That said we released a patch for 2.2 this morning with 2.2.1, so you  can just try again with that instead and see if it works :) (although this won't help with the second error I expect)
